templates/tfengine/components/audit/variables.tf

```diff
@@ -54,15 +36,43 @@ variable "logs_bigquery_dataset" {
     dataset_id = string
     sink_name = string
   })
-  description = {{schemaDescription $props.logs_bigquery_dataset.description}}
+  description = <<EOF
+    {{$props.logs_bigquery_dataset.description}}
+
+    * dataset_id = {{$logsBigQueryProps.dataset_id.description}}
```

Comment:
Can you have this part iterate through `$logsBigQueryProps` and get the key names that way? Seems more robust.

Also nit, but add "Fields:" right above so it's obvious what this list is, like
```
Fields:
* name = ...
* sink_name = ...
```

Reply:
I was thinking about it, but automating at this point may become difficult since the schema is not that consistent.

Two main problems I see:

1. Sometimes the properties are in `items` key. I don't really see a way to know if iterating through properties or items. Aside from that, I'm not aware of other special cases.

2. There are schema variables that don't become a terraform variable. Sometimes they just don't appear or their name is changed.

I'd suggest to better document this in an issue and fix once every recipe update is stable

Reply:
Oh, those are good points. Then just keep it like this but add the header still.

Reply:
Do you mean the `TODO()` header?

Reply:
No, I mean a line saying "Fields:" (or something similar) above the list of fields/properties.

Reply:
Ok, got it.

Added